lib/matplotlib/tests/test_axes.py

```diff
@@ -3155,7 +3155,7 @@ def _bxp_test_helper(
     logstats = mpl.cbook.boxplot_stats(
         np.random.lognormal(mean=1.25, sigma=1., size=(37, 4)), **stats_kwargs)
     fig, ax = plt.subplots()
-    if bxp_kwargs.get('vert', True):
+    if not bxp_kwargs.get('orientation') or bxp_kwargs.get('orientation') == 'vertical':
```

Comment:
```suggestion
    if bxp_kwargs.get('orientation', 'vertical') == 'vertical':
```
Defines a default and so avoids calling `get` twice.